src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,154 @@
+package duke;
+
+/**
+ * Parses user inputs or saved strings.
+ */
+public class Parser {
+    private static boolean isBye(String s) {
+        return s.toLowerCase().equals("bye");
+    }
+
+    private static boolean isList(String s) {
+        return s.toLowerCase().equals("list");
+    }
+
+    private static boolean isDone(String s) {
+        return s.length() >= 4 && s.substring(0, 4).toLowerCase().equals("done");
```

Comment:
Would be good to place line break to remove readability

Reply:
remove readability, do you mean improve readability, kendrew chan?

Reply:
Fixed similar parts.